crates/gossip/src/gossip.rs

```diff
@@ -552,11 +571,17 @@ mod test {
                 .await
                 .unwrap();
 
+            let peer_meta_store = K2PeerMetaStore::new(
```

Comment:
This is a stateless wrapper, so it's safe to construct a "shadow" one here for now